lib/driver.js

```diff
@@ -171,7 +171,9 @@ class EspressoDriver extends BaseDriver {
       return [sessionId, caps];
     } catch (e) {
       await this.deleteSession();
-      throw e;
+      e.message += '. If the cause might be espresso driver or server, ' +
```

Comment:
I would rather change the text to `Check https://github.com/appium/appium-espresso-driver#contributing regarding advanced session startup troubleshooting.`